tests/compute/groupshared.slang

```diff
@@ -1,39 +1,32 @@
 // groupshared.slang
-
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute -dx12
 //TEST(compute, vulkan):COMPARE_COMPUTE_EX:-vk -compute
 //TEST(compute):COMPARE_COMPUTE_EX:-cuda -compute
-
 //TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):out, name=gBuffer
 RWStructuredBuffer<int> gBuffer;
-
-#define THREAD_COUNT 4
-
+static const int A = 4;
+static const int B = 1;
+static const int THREAD_COUNT = A + B;
```

Comment:
Shouldn't this be `A * B` so that the thread count is the same as in the original?